### ISSUE TITLE
uploading empty files

### ISSUE THREAD

**Issue by citoki** [pfefferf/grunt-nwabap-ui5uploader#5]
Hi Florian,

by accident I wanted to upload an empty file and got the following error back:
```sh
Running "nwabap_ui5uploader:upload_build" (nwabap_ui5uploader) task
Fatal error: Argument error, options.body.
```
Is this a restriction by ABAP/ADT?

Maybe it's possible to provide a more verbose error message, which filename could not be uploaded. In fact some files where successfully uploaded. It would be great to see on each HTTP OK response, which file is transferred completely.

Cheers
--
Steffen

**Comment by pfefferf** [comment]
Hi,

can you share your config for a detailed check.

Thx.

**Comment by citoki** [comment]
Gruntfile.js:

```js
    // command line arguments for uploadBuild tasks
    var sGatewayUser = grunt.option("user");
    var sGatewayPassword = grunt.option("pwd");
    var sTransportRequest = grunt.option("tr");

        "nwabap_ui5uploader": {
            options: {
                conn: {
                    server: "http://gateway.server.intra.net:8010",
                    useStrictSSL: false
                },
                auth: {
                    user: sGatewayUser,
                    pwd: sGatewayPassword
                }
            },
            "upload_build": {
                options: {
                    ui5: {
                        transportno: sTransportRequest,
                        package: "ZPACKAGE",
                        bspcontainer: "ZUI5_APP",
                        "bspcontainer_text": "UI5 app"
                    },
                    resources: {
                        cwd: "build/release",
                        src: "**/*.*"
                    }
                }
            }
        }

    grunt.registerTask("uploadBuild", ["nwabap_ui5uploader"]);
```
I started this grunt task with following command:
`grunt uploadBuild --user=user1 --pwd=secret --tr=SIDK900053`

**Comment by pfefferf** [comment]
The crash was caused by an empty buffer object due to the empty file. Fixed with v0.1.10.

The "instant" log functionality for processed files is added to my list for implementation.

Thx.
Florian
